### ISSUE TITLE
debian-8.6 VMware hangs on graceful shutdown

### ISSUE THREAD

**Issue by jvf** [chef/bento#712]
I'm trying to use the `bento/debian-8.6` box from Atlas. Starting and sshing into the box work fine, but `vagrant halt` hangs forever in `default: Attempting graceful shutdown of VM...`. Using the `hashicorp/precise-64` box works fine. Any idea what the reason could be or how to debug and fix this? I'm happy to provide more information.

Environment:
OS: Mac OS X 10.11.6
Vagrant 1.8.6
vagrant-vmware-fusion (4.0.14)
VMware Fusion Pro 8.5.1

**Comment by jvf** [comment]
Running `VAGRANT_LOG=debug vagrant halt` gives:

```
...
 INFO guest: Detected: debian!
DEBUG guest: Searching for cap: halt
DEBUG guest: Checking in: debian
DEBUG guest: Checking in: linux
DEBUG guest: Found cap: halt in linux
 INFO guest: Execute capability: halt [#<Vagrant::Machine: default (HashiCorp::VagrantVMwarefusion::Provider)>] (debian)
DEBUG ssh: Re-using SSH connection.
 INFO ssh: Execute: shutdown -h now (sudo=true)
DEBUG ssh: stderr: stdin: is not a tty

DEBUG ssh: Sending SSH keep-alive...
DEBUG ssh: Sending SSH keep-alive...
...
```

The problem and fix is described [here] (https://github.com/mitchellh/vagrant/issues/6207#issuecomment-234332985). Installing `libpam-systemd` fixed the issue for me. I would still consider this potentially a bug in the Debian template, as the box should work correctly without installing additional software.

**Comment by cheeseplus** [comment]
Fixed by #693 - we just need to build the next version 2.3.1 and push.
